modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -77,7 +93,43 @@ definition {
 			locator1 = "TestrayManage#NEW_OPTION_NAME",
 			value1 = "${key_value}");
 
+		Select(
+			locator1 = "TestrayManage#SELECT_CATEGORIES_IN_OPTIONS",
+			value1 = "${categorie}");
+
 		Click(locator1 = "TestrayManage#SAVE_BUTTON");
+
+		CommerceUtilities.checkSuccessAlertAppearsAndDisappears();
+	}
+
+	macro deleteCategory {
+		TestrayLayout.clickManageOnSidebar();
+
+		TestrayManage.selectOptionsManage(itemValue = "Categories");
+
+		while (IsElementPresent(key_projectName = "Environments_", locator1 = "Testray#PROJECT_NAME")) {
+			TestrayContextMenu.clickContextMenuProjects(name = "Environments_");
+
+			TestrayContextMenu.selectItemFromProjectsContextMenu(item = "Delete");
+
+			AssertConfirm(value1 = "Are you sure you want to delete this item?");
+
+			Alert.viewSuccessMessage();
+
+			CommerceUtilities.checkSuccessAlertAppearsAndDisappears();
+		}
+	}
+
```

Comment:
Can we have this macro to accept an specific category to be deleted (similar to deleteOption) and another macro to delete all categories?